library/ubuntu

```diff
@@ -43,8 +43,8 @@ Tags: 20.04, focal-20191030, focal, devel
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
 Directory: focal
 
-# 20191107 (trusty)
-Tags: 14.04, trusty-20191107, trusty
+# 20190515 (trusty)
+Tags: 14.04, trusty-20190515, trusty
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le
 Directory: trusty
```

Comment:
Without the full set of commit hashes as well, this block still won't work (because it's still referring to the same commit); here's what we need to update this block to be instead:

```console
Tags: 14.04, trusty-20190515, trusty
Architectures: amd64, arm32v7, arm64v8, i386, ppc64le
GitCommit: 7a1a4f883de2d81351a82c86c24a0033dcae03db
Directory: trusty
amd64-GitCommit: 010bf9649b1d10e2c34b159a9a9b338d0fdd4939
amd64-GitFetch: refs/heads/dist-amd64
arm32v7-GitCommit: b9a951ace1f2e2e418828a197f62678c337f903e
arm32v7-GitFetch: refs/heads/dist-arm32v7
arm64v8-GitCommit: dffe0ea71413c9ca297a0935bade25c3ad3348a3
arm64v8-GitFetch: refs/heads/dist-arm64v8
i386-GitCommit: 2f8e166f4a725c5bcf3e96e2986d312f39a14098
i386-GitFetch: refs/heads/dist-i386
ppc64le-GitCommit: bbbf38c2900bd6aea408e3690be45e74bf2de1a5
ppc64le-GitFetch: refs/heads/dist-ppc64le
s390x-GitCommit: 4f55bf693232c85b18f9e6084c8f6780a24c3ba4
s390x-GitFetch: refs/heads/dist-s390x
```